### PR TITLE
Add doc on policy check enablement and clean-up P&T release notes

### PR DESCRIPTION
**Pull request by douglas-reid**
Also, includes deprecation notice for in-proc adapters in release notes (#3670).

Fixes: #3671

### CODE REVIEW

File: content/docs/tasks/policy-enforcement/enabling-policy/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+
+This task shows you how to enable Istio policy enforcement.
+
+## At install time
+
+In the default Istio installation profile, policy enforcement is disabled. To install Istio
+with policy enforcement on, use the `--set global.disablePolicyChecks=false` Helm install option.
+
+Alternatively, you may [install Istio using the demo profile](https://preliminary.istio.io/docs/setup/kubernetes/install/kubernetes/),
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/enabling-policy/index.md]: Remove "https://preliminary.istio.io", shoud just be /docs/...

**Comment by kyessenov** [comment]
Does this really work?
Don't you need to restart pilot (at the very least)?

**Comment by douglas-reid** [comment]
@kyessenov I tested this in a cluster and was able to see `Check()` calls without restarting Pilot.

**Comment by kyessenov** [comment]
Hmm, that didn't work for me. But I guess it might just take time.

**Comment by douglas-reid** [comment]
@kyessenov ok. can you check your test cluster after a minute or two? i was verifying by checking the metrics on `istio-policy` pods directly.  but I don't recall the time between testing the commands to update the config and actually sending traffic.

maybe we need to add a "wait 2 minutes" step?

**Comment by kyessenov** [comment]
We should get to the bottom of this, and figure out if this is even supposed to work. Last time I asked, I was told configmaps don't auto-update in pilot.

**Comment by kyessenov** [comment]
What you might be seeing is that some other update event triggers configmap reload. But it doesn't guarantee that configmaps themselves trigger a reload.

**Pull request merged**